otter/models/cass.py

```diff
@@ -443,11 +443,27 @@ def _jsonloads_data(raw_data):
     return data
 
 
+def _group_status(status, deleting):
+    if deleting:
+        return ScalingGroupStatus.DELETING
+    else:
+        # TODO: #1304
+        if status is None:
+            return ScalingGroupStatus.ACTIVE
+        elif status == 'DISABLED':
+            return ScalingGroupStatus.ERROR
+        else:
+            return ScalingGroupStatus.lookupByName(status)
+
+
 def _unmarshal_state(state_dict):
     desired_capacity = state_dict['desired']
     if desired_capacity is None:
         desired_capacity = 0
 
+    status = _group_status(state_dict['status'],
+                           state_dict.get('deleting', False))
```

Comment:
radix:  I think the three places that `_unmarshal_state` is called are in `view_manifest`, `view_state`, and `list_scaling_group_states`.  `view_manifest` does query for `deleting` and include it in the response, but I think `view_state` and `list_scaling_group_states` query the index for groups that are not deleting.  So it doesn't have `deleting` in the response, but `deleting` can be assumed to be `False`.

It is kinda hard to tell that though without digging through all the calls to see what the CQL is though, so yeah, adding `deleting` may be more explicit.